tests/d3d12.c

```diff
@@ -37730,7 +37752,7 @@ static void test_register_space(bool use_dxil)
     for (i = 0; i < 4; i++)
     {
         /* Start value of 9 is for the StructuredBuffer we write to. */
-        float reference = 2 * 3 * 4 * 5 * 6 * 7 * 8 * 10 * 11;
+        float reference = 2 * 3 * 4 * 5 * 6 * 7 * 8 * (10 * 10) * 11;
```

Comment:
10 comes from the additional texture lookup?

Reply:
Yes. Kinda messy to do things like this though, but it's basically a hardcoded value either way.